src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandler.java

```diff
@@ -172,9 +173,31 @@ protected void handleAudioVideoDocuments(SscsCaseData sscsCaseData) {
         validateDwpResponseDocuments(sscsCaseData, preSubmitCallbackResponse);
 
         validateDwpAudioVideoEvidence(sscsCaseData, preSubmitCallbackResponse);
+
+        validateSscs5CaseConfidentialityNoAppendix12Doc(sscsCaseData, preSubmitCallbackResponse);
+
         return preSubmitCallbackResponse;
     }
 
+
+    private void validateSscs5CaseConfidentialityNoAppendix12Doc(SscsCaseData sscsCaseData, PreSubmitCallbackResponse<SscsCaseData> preSubmitCallbackResponse) {
+        if (Benefit.TAX_CREDIT.getShortName().equals(sscsCaseData.getAppeal().getBenefitType().getCode())
+                || Benefit.GUARDIANS_ALLOWANCE.getShortName().equals(sscsCaseData.getAppeal().getBenefitType().getCode())
+                || Benefit.TAX_FREE_CHILDCARE.getShortName().equals(sscsCaseData.getAppeal().getBenefitType().getCode())
+                || Benefit.HOME_RESPONSIBILITIES_PROTECTION.getShortName().equals(sscsCaseData.getAppeal().getBenefitType().getCode())
+                || Benefit.CHILD_BENEFIT.getShortName().equals(sscsCaseData.getAppeal().getBenefitType().getCode())
+                || Benefit.THIRTY_HOURS_FREE_CHILDCARE.getShortName().equals(sscsCaseData.getAppeal().getBenefitType().getCode())
+                || Benefit.GUARANTEED_MINIMUM_PENSION.getShortName().equals(sscsCaseData.getAppeal().getBenefitType().getCode())
+                || Benefit.NATIONAL_INSURANCE_CREDITS.getShortName().equals(sscsCaseData.getAppeal().getBenefitType().getCode())) {
+            if (StringUtils.equalsIgnoreCase(sscsCaseData.getDwpEditedEvidenceReason(), "childSupportConfidentiality")) {
+                if (sscsCaseData.getAppendix12Doc() != null && sscsCaseData.getAppendix12Doc().getDocumentLink() != null) {
+                    preSubmitCallbackResponse.addError(APPENDIX_12_DOC_NOT_FOR_SSCS5_CONFIDENTIALITY);
```

Comment:
You don't actually need this change in the About to Submit handler and the Midevent handler - please just have it in the mid event otherwise we are duplicating code